shell/platform/windows/angle_surface_manager.cc

```diff
@@ -20,25 +20,31 @@ AngleSurfaceManager::~AngleSurfaceManager() {
   CleanUp();
 }
 
-bool AngleSurfaceManager::InitializeEGL(const EGLint* attributes) {
+static void logEglError(std::string message) {
+  EGLint error = eglGetError();
+  std::cerr << message << std::endl;
+  std::cerr << "eglGetError returned " << error << std::endl;
+}
+
+bool AngleSurfaceManager::InitializeEGL(const std::pair<const std::string, const EGLint*> config) {
   PFNEGLGETPLATFORMDISPLAYEXTPROC eglGetPlatformDisplayEXT =
       reinterpret_cast<PFNEGLGETPLATFORMDISPLAYEXTPROC>(
           eglGetProcAddress("eglGetPlatformDisplayEXT"));
   if (!eglGetPlatformDisplayEXT) {
-    std::cerr << "EGL: eglGetPlatformDisplayEXT not available" << std::endl;
+    logEglError("EGL: eglGetPlatformDisplayEXT not available");
     return false;
   }
 
   egl_display_ = eglGetPlatformDisplayEXT(EGL_PLATFORM_ANGLE_ANGLE,
-                                          EGL_DEFAULT_DISPLAY, attributes);
+                                          EGL_DEFAULT_DISPLAY, config.second);
 
   if (egl_display_ == EGL_NO_DISPLAY) {
-    std::cerr << "EGL: Failed to get a compatible EGLdisplay" << std::endl;
+    logEglError("EGL: Failed to get a compatible EGLdisplay");
     return false;
   }
 
   if (eglInitialize(egl_display_, nullptr, nullptr) == EGL_FALSE) {
-    std::cerr << "EGL: Failed to initialize";
+    logEglError("EGL: Failed to initialize: " + config.first);;
```

Comment:
I just realized that this error message is going to print one or more error messages when we successfully do fallback, which is against Flutter's guidance about only printing error messages that are actionable (since there's no action to be taken on successful fallback).

We should only print this on the last attempt, rather than each one.